modular_skyrat/code/_HELPERS/roundend.dm

```diff
@@ -0,0 +1,27 @@
+/datum/controller/subsystem/ticker/declare_completion()
+	..()
+	for(var/M in GLOB.player_list)
+		var/mob/m = M
+		if (!m.client)
+			continue
+		m.client.verbs += /client/proc/eorg_teleport
+		if (m.client && m.client.prefs && m.client.prefs.eorg_teleport)
```

Comment:
This m.client check is redundant as you have already checked for it above.

Reply:
For some reason it didn't work beforehand without it, although that might be due to something else.
If you want me to remove the second m.client I can, it's just going to need another bunch of tests, spleh.